testgrid/cmd/configurator/prow.go

```diff
@@ -0,0 +1,116 @@
+/*
```

Comment:
Super interested in making sure people don't try and add an entire TestGrid config in annotations; not sure where's best to enforce that, or if it's super-enforceable (beyond being careful in reviewing TestGrid code).

Maybe a note like this PR's first comment somewhere in docs?

Reply:
I added a note to the list of annotations directing people to you if they are thinking about adding more.